strawberry/http/json_encoder.py

```diff
@@ -0,0 +1,11 @@
+import json
+from typing import Callable
+
+from . import GraphQLHTTPResponse
+
+
+JsonEncoder = Callable[[GraphQLHTTPResponse], str]
```

Comment:
How about prefixing this with `T` to indicate that it's a type annotation? `TJsonEncoder` (or `TJsonSerializer` if we change the name)